README.md

```diff
@@ -28,6 +28,8 @@ However, storybook will filter all the plugins for webpack according to internal
 }
 ```
 
+Note that only plugins defined with [webpack-chain](https://cli.vuejs.org/guide/webpack.html#chaining-advanced) config can be used in storybook.
```

Comment:
We need to revisit this. I don't think this line is valid anymore.

Reply:
Right, pushed